dair_pll/drake_utils.py

```diff
@@ -21,10 +21,15 @@
 from dataclasses import dataclass, field
 from typing import Tuple, Dict, List, Optional, Mapping, cast, Union, Type
 
+import numpy as np
+import os
+import os.path as op
+import pdb
```

Comment:
`pdb` imports should be left out of main branch

Reply:
Noted.  Removed.